README.md

```diff
@@ -237,4 +237,4 @@ Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhan
 
 ## License
 
-The Charmed Kafka K8s Operator is free software, distributed under the Apache Software License, version 2.0. For more information, see the [LICENSE](https://github.com/canonical/kafka-k8s-operator/blob/main/LICENSE) file .
+The Charmed Apache Kafka K8s Operator is free software, distributed under the Apache Software License, version 2.0. For more information, see the [LICENSE](https://github.com/canonical/kafka-k8s-operator/blob/main/LICENSE) file .
```

Comment:
```suggestion
The Charmed Apache Kafka K8s is free software, distributed under the Apache Software License, version 2.0. For more information, see the [LICENSE](https://github.com/canonical/kafka-k8s-operator/blob/main/LICENSE) file .
```

Reply:
```suggestion
Charmed Apache Kafka K8s is a free software, distributed under the Apache Software License, version 2.0. For more information, see the [LICENSE](https://github.com/canonical/kafka-k8s-operator/blob/main/LICENSE) file.
```
Note: This update in wording changes the meaning quite a bit. Previously, we talked about the operator. Now we are talking about the whole solution, including Apache Kafka. It has the same Licence, so I guess it's OK. But you should be aware of the gravity of this change. @deusebio